models/DataObject/AbstractObject.php

```diff
@@ -115,6 +116,19 @@ abstract class AbstractObject extends Model\Element\AbstractElement
      */
     protected ?string $childrenSortOrder = null;
 
+    /**
+     * @internal
+     *
+     * @var string|null
+     */
+    protected $classId = null;
```

Comment:
Use native types instead of php doc.

```suggestion
    /**
     * @internal
     */
    protected ?string $classId = null;
```